src/Writer/Extension/GooglePlayPodcast/Entry.php

```diff
@@ -66,10 +66,13 @@ public function getEncoding()
     /**
      * Set a block value of "yes" or "no". You may also set an empty string.
      *
-     * @param  string
+     * @param string
+     *
      * @throws Writer\Exception\InvalidArgumentException
+     *
+     * @return void
      */
-    public function setPlayPodcastBlock($value)
+    public function setPlayPodcastBlock($value): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.